### ISSUE TITLE
 "go build github.com/ibm-messaging/mq-golang-jms20: no non-test Go files.." while getting package by go get

### ISSUE THREAD

**Issue by VladimirDe** [ibm-messaging/mq-golang-jms20#14]
Error with go get github.com/ibm-messaging/mq-golang-jms20@v1.2.0 through any proxy (gocenter.io, proxy.golang.org)
```
 go version go1.15.2 linux/amd64
  GO111MODULE="on"
  GOARCH="amd64"
  GOBIN=""
  GOCACHE="/root/.cache/go-build"
  GOENV="/root/.config/go/env"
  GOEXE=""
  GOFLAGS=""
  GOHOSTARCH="amd64"
  GOHOSTOS="linux"
  GOINSECURE=""
  GOMODCACHE="/root/go/pkg/mod"
  GONOPROXY=""
  GONOSUMDB="*"
  GOOS="linux"
  GOPATH="/go"
  GOPRIVATE=""
  GOPROXY="https://gocenter.io"
  GOROOT="/go-1.15.2"
  GOSUMDB="sum.golang.org"
  GOTMPDIR=""
  GOTOOLDIR="/go-1.15.2/pkg/tool/linux_amd64"
  GCCGO="gccgo"
 go get -v -u github.com/ibm-messaging/mq-golang-jms20
 go build github.com/ibm-messaging/mq-golang-jms20: no non-test Go files in /go/pkg/mod/github.com/ibm-messaging/mq-golang-jms20@v1.2.0
```

**Comment by matrober-uk** [comment]
Hi @VladimirDe - thanks for your issue report!

There doesn't seem to be a "good" way of suppressing this error in the case where there are deliberately not expected to be any non-test Go files in the root of the repo, so I've merged a PR #15 to adopt one of the common workarounds as described at https://github.com/golang/go/issues/29268#issuecomment-625978368

Please can you give it a try in your scenario and close out the issue if you are happy it resolves the problem, or add a comment if you find it causes unexpected side effects elsewhere.

Regards,

Matt

**Comment by VladimirDe** [comment]
Thank you very much !!! Actually could do it myself (so simple) !!!! I close the issue
